### PR TITLE
adjusts for test and implemented in QasBox

### LINKED ISSUE #87: Box Test

### PR DESCRIPTION
**Pull request by DouglasCalora**
Closes #87

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "npx jest",
+    "test-watch": "jest --watch",
```
Comment (caiotarifa) [inline ui/package.json:8]: Mudar para `test:watch`

File: ui/src/components/box/QasBox.spec.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+describe('Test QasBox component', () => {
+  const wrapper = mount(QasBox)
+
+  it('Mount component', async () => {
+    expect(wrapper.exists()).toBe(true)
+    expect(wrapper.classes()).toContain('qas-box__elevated')
+  })
+
+  it('Set form-mode', async () => {
```
Comment (caiotarifa) [inline ui/src/components/box/QasBox.spec.js:14]: `display mode as form`

**Review by caiotarifa — APPROVED** [review]

File: ui/package.json (hunk 1 above)
Comment (DouglasCalora) [reply ui/package.json]: feito

File: ui/src/components/box/QasBox.spec.js (hunk 2 above)
Comment (DouglasCalora) [reply ui/src/components/box/QasBox.spec.js]: feito

**Pull request merged**